### DISCUSSION TITLE
Enum error while running simu5g simulation

### DISCUSSION THREAD

**Discussion by ErolKocoglu** [omnetpp/omnetpp#1287] [Q&A] [answered]
![Ekran Görüntüsü (842)](https://github.com/user-attachments/assets/0800b9c3-e300-4c3b-ba58-4f1f9a713b12)
I am using some elements (Ue, gnodeB, etc.) from simu5g. When trying to run the simulation, I am getting this error. It says the enum property is only allowed on string parameters. How can I fix this?

**Comment by rhornig** [comment, marked as answer]
simu5g 1.2.2 is not compatible with omnetpp 6.1.

> Edit: Simu5G 1.2.3 and 1.3.0 is out now and these versions support omnetpp 6.1 properly.

**Reply by ErolKocoglu** [reply]
Thank you. I installed 6.0.3 and now it works.
